ibrcommon/tests/net/tcpstreamtest.cpp

```diff
@@ -89,8 +91,10 @@ void tcpstreamtest::runTest()
 	} catch (const ibrcommon::vsocket_interrupt &e) {
 		// select interrupted
 	} catch (const ibrcommon::socket_exception &e) {
-		CPPUNIT_FAIL(std::string("client error: ") + e.what());
+		std::cerr << "client error: " << + e.what() << std::endl;
```

Comment:
CPPUNIT_FAIL here results in main thread termination but listening thread to keep waiting unjoined. To if the client connection fails, the test hangs indefinitely. With this change it fails fast.